CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # GraphDB Helm chart release notes
 
+## Version 11.1.3
```

Comment:
Rebase the branch on top of `PRO-444-features-for-11-2` and include this for `11.2.0` in the changelog. We don't want to release a patch version for a minor change in the readme.